### PR TITLE
created main window composite

### LINKED ISSUE #70: App Bar
- [*] Allows to edit the project title;
- [*] Has an import button with event handler
- [*] Has a reset project button with event handler
- [*] Has a save/download button with event handler
- [*] Has a project settings button with event handler

### PR DESCRIPTION
**Pull request by adam-digirati**
closes #70

### CODE REVIEW

File: src/components/VariationsMainView/VariationsMainView.mdx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: Variations Main View 
+menu: Components
```
Comment (stephenwf) [inline src/components/VariationsMainView/VariationsMainView.mdx]: Could we put this under proofs of concepts, or maybe replace the material design view, its very similar.

File: src/components/VariationsMainView/VariationsMainView.js [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+} from '@material-ui/core';
+import {
+  Info,
+  AddCircle,
+  Delete,
+  CloudDownload,
+  Settings,
+} from '@material-ui/icons';
+
+const TempAppBar = props => (
```
Comment (stephenwf) [inline src/components/VariationsMainView/VariationsMainView.js]: Think we need a ticket for the app bar?

File: src/components/VariationsMainView/VariationsMainView.mdx (hunk 1 above)
Comment (adam-digirati) [reply src/components/VariationsMainView/VariationsMainView.mdx]: done https://github.com/digirati-co-uk/timeliner/pull/67/commits/294f139f3297c19c0f215a4244c752e81a36f24a

File: src/components/VariationsMainView/VariationsMainView.js (hunk 2 above)
Comment (adam-digirati) [reply src/components/VariationsMainView/VariationsMainView.js]: Created the ticket #70, refactored the component

**Pull request merged**